robot.py

```diff
@@ -58,8 +58,8 @@ def createObjects(self):
         self.shooter_outer_motor = rev.CANSparkMax(2, rev.MotorType.kBrushless)
 
         self.colour_sensor = rev.color.ColorSensorV3(wpilib.I2C.Port.kOnboard)
-        self.spinner_motor = wpilib.Spark(2)
-        self.spinner_solenoid = wpilib.DoubleSolenoid(2, 3)
+        self.spinner_motor = wpilib.Spark(4)
+        self.spinner_solenoid = wpilib.Solenoid(2, 3)
```

Comment:
From my understanding, we don't have a PCM with ID 2, and don't plan to… are you sure this is correct?

Reply:
Fixed